api/src/server_events.h

```diff
@@ -31,10 +31,6 @@
 extern "C" {
 #endif
 
-// forward-declare to avoid circular dependency of including registered_entity_iterator.h
-typedef struct _RegisteredEntityIterator RegisteredEntityIterator;
```

Comment:
The problem is that this has meant that you now return the struct _RegisteredEntityIterator instead of hiding it